frontend/server/src/test/resources/management_open_api.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "TorchServe APIs",
     "description": "TorchServe is a flexible and easy to use tool for serving deep learning models",
-    "version": "%s"
+    "version": "0.1.1"
```

Comment:
Why are we changing this to a hard-coded value? We will need to update the expected open-api json every time we update the version. We already have UTs to ensure that the versions are correctly reflected everywhere.

Reply:
Reverted